lib/validation.rb

```diff
@@ -113,6 +113,14 @@ def self.validate_postgres_size(size)
     postgres_size
   end
 
+  def self.validate_postgres_storage_size(size, storage_size)
+    storage_size = storage_size.to_i
+    pg_size = validate_postgres_size(size)
+    allowed_sizes = (pg_size.min_storage_size_gib..pg_size.max_storage_size_gib).step(pg_size.storage_size_step_gib)
```

Comment:
if we create the PostgresSize struct with this array provided already, we don't need to programmatically calculate this here.